### ISSUE TITLE
get --expand doesn't work properly

### ISSUE THREAD

**Issue by jwheare** [stripe/stripe-cli#47]
`-d 'expand[]=foo'` works to expand fields in API responses but `--expand foo` (as per documentation) gives the following error:
```
{
  "error": {
    "message": "Invalid array",
    "param": "expand",
    "type": "invalid_request_error"
  }
}
```

Tried with:
```
> stripe get subscriptions/sub_FSBap8m2DfyHl8 --expand latest_invoice
> stripe get subscriptions/sub_FSBap8m2DfyHl8 -d 'expand[]=latest_invoice'
```

### Environment
macOS 10.14.5, stripe CLI installed via homebrew
```
> stripe --version
stripe version 0.2.2
```

**Comment by jwheare** [comment]
Nvm, seems fixed in 0.2.3 #33

**Comment by ob-stripe** [comment]
Yes, should be fixed already! Thanks for the report anyway :)
